### DISCUSSION TITLE
part-ii/2019-143/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#267] [Question comments]
Part II 2019 Statistical Physics: Paper 1, Section II, D 

https://questions.tripos.org/part-ii/2019-143/

**Comment by thenewtonium** [comment]
<details><summary>Part (a)</summary>

This is a very confusingly worded question; it makes a lot more sense to define efficiency *before* showing that an isotherms+adiabats cycle is most effective.

To calculate the efficiency in terms of temperature, the method used in Tong's notes is not very helpful here. Instead, we note that since the net work done *by* the system is equal to the net heat *supplied* to the system over the cycle, the net work done by the system is $\oint T dS$ i.e. the area enclosed by the cycle in the $(T,S)$-plane (by Green's thm / thinking about it). So the net work done by the system is $W = (T_2-T_1)(S_2-S_1)$ where $S_2 > S_1$ are the entropies of the system at the adiabatic steps.

On the other hand, heat is only supplied to the system for $dS > 0$. Integrating $TdS$ along the isotherm with increasing $S$ yields $Q_H = T_2(S_2 - S_1)$.
Thus we get the usual efficiency formula $\eta = W / Q_H = 1 - \frac{T_1}{T_2}$.

</details>

<details><summary>Part (b)</summary>

Note that $S$ is not necessarily continuous! Instead we need to use that the *latent heat* – which is defined as $T_c$ times the size of the discontinuity in $S$ as you pass through $T_c$ – is experimentally measurable (e.g. by putting the system just below $T_c$, and measuring the heat that needs to be supplied to raise its temperature just above $T_c$). Then combining with the third law ($S = 0$ at $T = 0$) would allow us to determine $\gamma$.
</details>
